dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -161,7 +165,10 @@ private static File findLocalRepository() {
     RepositorySystemSession session = RepositoryUtility.newSession(system);
 
     ArtifactDescriptorRequest request = new ArtifactDescriptorRequest();
-    request.addRepository(RepositoryUtility.CENTRAL);
+
+    for (RemoteRepository repository : mavenRepositories) {
```

Comment:
I was thinking to use addRepositoryToRequest here too. But `ArtifactDescriptorRequest` and `CollectRequest` don't share a class or interface. AS line 169 is within `RepositoryUtility`, I let this line access `mavenRepositories` directly.